packages/vite-plugin-nitro/src/lib/build-sitemap.ts

```diff
@@ -0,0 +1,73 @@
+import { SitemapConfig } from './options';
```

Comment:
nit: put local imports last

```suggestion
import { SitemapConfig } from './options';
```